demisto_sdk/__main__.py

```diff
@@ -79,20 +79,12 @@ def main(config, version, env_dir):
     '-h', '--help'
 )
 @click.option(
-    '--infile', '-i',
-    help='The yml file to extract from',
-    required=True
+    '-i', '--input', help='The yml file to extract from', required=True
```

Comment:
I think we agreed on `in` and `out`

Reply:
`in` is a saved word, so it is not possible.